packages/rollup-plugin-webbundle/README.md

```diff
@@ -235,9 +236,17 @@ const pluginOptionsWithCustomSigningStrategy = {
 };
 ```
 
+### `integrityBlockSign.strategies`
+
+Type: `Array<ISigningStrategy>`
+
+Use this overload to sign a bundle with multiple keys or to specify a custom
+`webBundleId`.
+
```

Comment:
also similarly to the integrityBlockSign.strategy, please add an example with this type of signing

Reply:
Done!